src/pages/index.js

```diff
@@ -69,7 +55,6 @@ const newPostCardModal = document.querySelector("#new-post-modal");
 const newPostCloseBtn = newPostCardModal.querySelector(".modal__close-btn");
 const newPostForm = newPostCardModal.querySelector(".modal__form");
 const submitBtn = newPostForm.querySelector(".modal__submit-btn");
```

Comment:
`submitBtn `

There are 3 modals with different buttons and forms in the project. You need to **name** variables and functions the way all could understand what they are: `profileForm`, `profileCloseButton` and so on


This is `addPostSubmitBtn`